sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md

```diff
@@ -85,6 +91,35 @@ You can also run tests via the command line using `dotnet test`, but that will r
 
 If you are using system environment variables, make sure to restart Visual Studio or the terminal after setting or changing the environment variables.
 
+## Test proxy
+
+Using the test proxy tool, migrate the local recording files to the external repository [azure-sdk-assets](https://github.com/Azure/azure-sdk-assets). 
```

Comment:
No migration needed anymore?

Reply:
assets.json has been added to the template, so  test framework will save recording files to `/.assets`, only we run `test-proxy push -a ./assets.json` to push local files to remote can pass ci  playback

Reply:
Yes, I mean we can remove this paragraph, there is no migration needed.
The next paragraph is talking about where the recording files are stored.